### DISCUSSION TITLE
r/city-china-shanghai-dianshanhu

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#58] [Announcements]
# r/city-china-shanghai-dianshanhu

游记｜淀山湖，上海周末露营度假好去处

https://blog.ops-coffee.cn/r/city-china-shanghai-dianshanhu

**Comment by beyond1314** [comment]
好的，来魔都好几年了，还没去过淀山湖。有时间按照楼主的路线开一圈。

**Comment by beyond1314** [comment]
楼主的户外群加入有什么条件吗？ 
一直看你的旅行文章，我都想加入了。
